node/src/runner.rs

```diff
@@ -64,13 +68,25 @@ pub async fn initialize(
         )?,
     );
 
+    // Mempool subsystem
+    let mempool = manager.add_subsystem(
+        "mempool",
+        mempool::make_mempool(
+            DummyMempoolChainState {},
+            chainstate.clone(),
+            mempool::pool::SystemClock {},
```

Comment:
Perhaps we should move the TimeGetter from the Chainstate crate and then use it for mempool too. We don't want to end up with 5 different time structs. Not a huge priority though. A TODO in here will do for now.

Reply:
Done, placed a TODO and opened an issue so we won't forget.